### ISSUE TITLE
CTRL+ENTER does not work when some SQL commands like SHOW / EXPLAIN / ANALYZE are in script without ";" before them

### ISSUE THREAD

**Issue by yoni-yad2** [dbeaver/dbeaver#33252]
### Description

Getting an error "SQL Error [1064] [42000]: You have an error in your SQL syntax...."  when trying to run using CTRL+ENTER any statement when SHOW / EXPLAIN / ANALYZE statements (there might be more) are not separated with ";" even though they are spaced with new lines between them.

### DBeaver Version

24.0.4.202405042322

### Operating System

Windows 10 64 bit

### Database and driver

RDS for MySQL v8.0.36

### Steps to reproduce

Try to run any statement separately with CTRL+ENTER in this script:

```
select 1

explain select 1

select 2

show variables like 'test'
```

**Comment by E1izabeth** [comment]
Please try 24.0.5 version or switch `Blank line is statement delimiter` setting to "Always"
![image](https://github.com/dbeaver/dbeaver/assets/28875055/d2ec4949-1518-467f-9741-542a65a5f01c)

**Comment by yoni-yad2** [comment]
Solved in v24.0.5.
